tinygrad/runtime/ops_triton.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+import hashlib
+import pycuda.driver as cuda # type: ignore
+from triton.compiler import compile as triton_compile
+from typing import Any
+from tinygrad.ops import  Compiled
+from tinygrad.runtime.ops_cuda import RawCUDABuffer
+from tinygrad.codegen.linearizer import LinearizerOptions
+from tinygrad.renderer.triton import uops_to_triton
+from tinygrad.helpers import DEBUG
+from tinygrad.runtime.ops_cuda import pretty_ptx 
+
+
+class TritonProgram:
```

Comment:
This should not be a runtime. Triton is a compiler.

Reply:
Okay. Will merge with ops_cuda